components/script/dom/domrectlist.rs

```diff
@@ -57,3 +66,18 @@ impl DOMRectListMethods for DOMRectList {
         self.Item(index)
     }
 }
+
+impl<'a> Iterator for Iter<'a> {
```

Comment:
Actually, the better question here is why not simply return `self.rects.iter()` in `fn iter`.